### PR TITLE
Configure and validate partitioning in the Storage module

### PR DESCRIPTION
**Pull request by poncovka**
Add support for storage configuration and validation in the Storage module.
It is implemented for the automatic and manual partitioning.

### CODE REVIEW

File: pyanaconda/modules/storage/partitioning/configure.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+        :param storage: an instance of Blivet
+        :param partitioning: a partitioning executor
+        """
+        super().__init__()
+        self._storage = storage
+        self._partitioning = partitioning
+
+    @property
+    def name(self):
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/partitioning/configure.py]: I know it is self describing but could you please add the documentation.

File: pyanaconda/storage/execution.py [hunk 2]
```diff
@@ -70,31 +72,35 @@ def do_kickstart_storage(storage, data):
...
+    """Get the executor of the enabled partitioning.
+
+    :return: an partitioning executor
+    """
+    if STORAGE.get_proxy(AUTO_PARTITIONING).Enabled:
+        return AutomaticPartitioningExecutor()
+
+    elif STORAGE.get_proxy(MANUAL_PARTITIONING).Enabled:
+        return ManualPartitioningExecutor()
+
```
Comment (jkonecny12) [inline pyanaconda/storage/execution.py]: Please remove these blank lines.

File: pyanaconda/modules/storage/partitioning/validate.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    def __init__(self, storage):
+        """Create a task.
+
+        :param storage: an instance of Blivet
+        """
+        super().__init__()
+        self._storage = storage
+
+    @property
+    def name(self):
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/partitioning/validate.py]: The same here.

File: pyanaconda/modules/storage/partitioning/configure.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/modules/storage/partitioning/configure.py]: Fixed.

File: pyanaconda/modules/storage/partitioning/validate.py (hunk 3 above)
Comment (poncovka) [reply pyanaconda/modules/storage/partitioning/validate.py]: Fixed.

File: pyanaconda/storage/execution.py (hunk 2 above)
Comment (poncovka) [reply pyanaconda/storage/execution.py]: Fixed.

**Comment by poncovka** [comment]
Updated.

**Comment by jkonecny12** [comment]
Looks even better now. Thanks!

**Pull request merged**